### DISCUSSION TITLE
Создание документа

### DISCUSSION THREAD

**Discussion by Leo-ui-528** [totumonline/totum-ru-issues-and-discussions#246] [Как это сделать?] [answered]
Здравствуйте, подскажите пожалуйста, как создать 1 таблицу-документ из введенных данных в другой их потом несколько. Пробовал ярлыками не получается.

**Comment by ttm-alexeypolunin** [comment]
Точно не Ярлыками. Опишите подробнее что у вас есть (таблицы, сколько, какие в них основные поля) и что нужно получить (таблицу вида пример)?

**Comment by Leo-ui-528** [comment, marked as answer]
Есть около 12 таблиц(город, округ, договор, поля наименование в них в основном) и 1(документ) центральная таблица,  после нажатия кнопки создать(на главной таблице документа) открывается  таблица в которую собираются данные ее заполняют с помощью селектов на другие или просто поля цифры, строки и дата, эти поля находятся в футере .  После чего, нажав кнопку сохранить нужно получить готовый документ желательно в строчном виде в главной таблице, где была кнопка создать и там же по уже созданным документам нужно создать поиск.

**Reply by erpic-ru** [reply]
Таблица циклов и в ней расчетная в цикле. После создания всех полей-селектов внутри цикла расчетной таблицы - нужно вытащить их ярлыками "наверх" в таблицу циклов. 

https://www.youtube.com/watch?v=bBnoGeNja9M

**Reply by ttm-alexeypolunin** [reply]
Дополню, что общий принцип тотума — пошаговое сохранение. Те при изменении поля значение записывается сразу и в таблице циклов наверху оно изменится сразу.

**Reply by ttm-alexeypolunin** [reply]
И вот еще про циклы, поднятие данных и поиск: https://github.com/totumonline/totum-ru-issues-and-discussions/discussions/75

**Comment by Leo-ui-528** [comment]
Спасибо
